<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CareSale</class>
 <widget class="QMainWindow" name="CareSale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>491</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>854</width>
    <height>491</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Care Sale helper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(144, 144, 216);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Greetings_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>661</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(0, 170, 170);
	border: 2px solid white;
	color: white
}</string>
    </property>
    <property name="text">
     <string>Добро пожаловать в помощника по продажам машин!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bmw_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>material/bmw_PNG1710.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cars_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}</string>
    </property>
    <property name="text">
     <string>Просмотр БД автомобилей:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	background-color: rgb(243, 162, 0);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Автоменеджер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	background-color: rgb(165, 0, 165);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Покупатель</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
